plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb

```diff
@@ -127,6 +128,10 @@ def set_replica_target
       template[:spec][:replicas] = @doc.replica_target if template[:kind] == 'Deployment'
     end
 
+    def set_name
+      template[:metadata][:name] = @doc.kubernetes_role.resource_name
```

Comment:
yes!

On Wed, Jun 29, 2016 at 8:45 AM, rata <notifications@github.com> wrote:

> In plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb
> <https://github.com/zendesk/samson/pull/1110#discussion_r68972320>:
>
> > @@ -127,6 +128,10 @@ def set_replica_target
> >        template[:spec][:replicas] = @doc.replica_target if template[:kind] == 'Deployment'
> >      end
> >
> > +    def set_name
> > +      template[:metadata][:name] = @doc.kubernetes_role.resource_name
>
> This changes the name used in deployment.spec.template.metadata.name in
> the yaml given to kubernetes? So, basically, it's override what the user
> configures there?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1110/files/69e70907f8826375ada24e525c7a1777046671ee#r68972320>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ-anbUnMaUUKTFZ08Hb0L36atrV1ks5qQpMHgaJpZM4JAoD7>
> .
>